redbot/setup.py

```diff
@@ -347,7 +347,16 @@ async def remove_instance_interaction() -> None:
 
 
 @click.group(invoke_without_command=True)
-@click.option("--debug", "--verbose", "-v", count=True)
+@click.option(
+    "--debug",
+    "--verbose",
+    "-v",
+    count=True,
+    help=(
+        "Increase the verbosity of the logs, each usage of this flag increases the verbosity"
+        " level by 1"
```

Comment:
Nitpick - this description is missing a period at the end where other help descriptions have one.